gcc4mbed/external/mbed/libraries/mbed/hal/sleep_api.h

```diff
@@ -18,7 +18,9 @@
 
 #include "device.h"
 
-#if DEVICE_SLEEP
+//#if DEVICE_SLEEP //TODO THIS WAS THE ORIGINAL BUT CONFLICTS WITH OTHER DEFINITION OF SLEEP
```

Comment:
Can you describe the problem that you were seeing which caused this change to be required? We should not be checking in anything to this part of the tree since it will be lost the next time we update GCC4MBED and the mbed SDK.